src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -138,23 +138,36 @@ const LabelCard: React.FC<LabelCardProps> = ({ data, index, quantity }) => {
 };
 
 export interface ShippingLabelsPdfProps {
+    data: ShippingLabelData[];
+}
+
+interface ShippingLabelsForSingleParcelProps {
     data: ShippingLabelData;
 }
 
+const ShippingLabelsForSingleParcel: React.FC<ShippingLabelsForSingleParcelProps> = ({ data }) => {
```

Comment:
I feel like this component should just take `numberOfLabels` as the only prop. All the other stuff in `data` is redundant.

Reply:
The component passes on the data to <SingleLabelCard> to display the data? Am I missing something?

Reply:
oh yes sorry my mistake, ignore me! Can we instead rename `data` to something more descriptive please?